src/blog/2022/04/flowforge-04-released.md

```diff
@@ -0,0 +1,97 @@
+---
+title: FlowForge 0.4 released
+subtitle: Getting ready for Node-RED 3.0
+description: Getting ready for Node-RED 3.0
+date: 2022-04-14 12:00:00.0
+authors: ["sam-machin"]
+---
+
+This release of the FlowForge add a seemingly small but significant new feature.
```

Comment:
Just a single small, and insignificant feature as opening line doesn't feel right.

Reply:
its says Significant NOT insignificant. Its meant to be a teaser to make you want to find out more.